Dockerfile

```diff
@@ -2,18 +2,18 @@ FROM registry.internal.logz.io:5000/node:14.16.0-alpine3.13 as js-builder
 
 WORKDIR /usr/src/app/
 
-COPY package.json yarn.lock ./
-COPY packages packages
+COPY grafana/package.json grafana/yarn.lock ./
```

Comment:
What is the reason of copying into `grafana` folder?

Reply:
to be able to build like grafana 8.5.1